### PR TITLE
[IMP] mail: introduce EmojiComponent

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2833579

### CODE REVIEW

File: addons/mail/static/src/components/emoji/emoji.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.Emoji" owl="1">
+        <span class="o_Emoji" t-on-click="ev => emojiListView.onClickEmoji(ev)" t-att-title="props.emoji.description" t-att-data-source="props.emoji.sources[0]" t-att-data-unicode="props.emoji.unicode" t-attf-class="{{ className }}" t-ref="root">
```
Comment (phenix-factory) [inline addons/mail/static/src/components/emoji/emoji.xml:5]: I don't understand the `onClick`  here, I don't think we need the arrow function anymore

**Review by phenix-factory — APPROVED** [review]
Small comment, otherwise LGTM

File: addons/mail/static/src/components/emoji/emoji.xml (hunk 1 above)
Comment (alexkuhn) [reply addons/mail/static/src/components/emoji/emoji.xml]: Changes done

**Comment by alexkuhn** [comment]
Moved to grouped PR: https://github.com/odoo/odoo/pull/90320

**Pull request closed without merging**